stdlib/adaml/_examples_/example.puml

```diff
@@ -0,0 +1,38 @@
+@startuml
+!include <AdaML/AdaML>
```

Comment:
Hi @theavege

- Could you fix the path?
  ↪ to be conform with your modified folder <adaml/AdaML>

_[FYI @arnaudroques]_
Regards,
Th.